src/cargo/core/compiler/compile_kind.rs

```diff
@@ -87,10 +87,10 @@ impl CompileKind {
             let deduplicated_targets = targets
                 .iter()
                 .map(|value| {
-                    // This neatly substitutes the manually-specified `host-tuple` target directive
+                    // This neatly substitutes the manually-specified `host-tuple` or `host` target directive
                     // with the compiling machine's target triple.
 
-                    if value.as_str() == "host-tuple" {
+                    if value.as_str() == "host-tuple" || value.as_str() == "host" {
```

Comment:
Thanks for the contribution. However, we don't want to support an extra `host` literal. Please remove